common/os_calls.c

```diff
@@ -2267,6 +2267,19 @@ g_strchr(const char* text, int c)
 }
 
 /*****************************************************************************/
+/* locates char in text */
```

Comment:
I notices that the purpose of these wrappers is to avoid using ```#include <string.h>``` and other standard headers, only this file with wrappers includes them and the rest of the xrdp's (but not sesman) sources use only wrappers.
I do not know why it is so, just followed the style.
Whole project could be refactored removing those wrappers.